README.md

```diff
@@ -1,32 +1,88 @@
-# Superformula Mobile Developer Coding Test
+# Superformula Mobile Test
 
-Make sure you read **all** of this document carefully, and follow the guidelines in it.
+[![Conventional Commits](https://img.shields.io/badge/Conventional%20Commits-1.0.0-green.svg)](https://conventionalcommits.org) 
 
-## Requirements
+[![macaco](https://img.shields.io/badge/Dart-0175C2?style=for-the-badge&logo=dart&logoColor=white)](https://img.shields.io/badge/Dart-0175C2?style=for-the-badge&logo=dart&logoColor=white) [![macaco](https://img.shields.io/badge/Flutter-02569B?style=for-the-badge&logo=flutter&logoColor=white)]() [![macaco](https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/felcesar/)
 
-There is only one test here currently, please review and get back to us.
+## Getting Started
 
-## What We Care About
+- Flutter version used in development `2.2.3`
+- Used null-safe wherever possible
 
-Use any libraries that you would normally use if this were a real production App. Please note: we're interested in your code & the way you solve the problem, not how well you can use a particular library or feature.
+### Building
+Building the project, run the following commands in the root dir
+(assuming you have a basic flutter & dart environment):
 
-_We're interested in your method and how you approach the problem just as much as we're interested in the end result._
+```bash
+$flutter pub get
+$flutter pub run build_runner [build/watch]
+```
+### Running
+You must have a valid device/simulator connected to get the device name. (You can also use VSCode Flutter extensions to make running and debugging through the GUI)
 
-Here's what you should strive for:
+```bash
+$flutter run -d [device_name]
+```
 
-- Good use of structure, security, and performance best practices.
-- Solid testing approach.
-- Extensible code.
```

Comment:
suggestion:  I don't see any widget tests.  Did I miss them or was this on purpose?